### PR TITLE
Issue #4165: Split and Organize Checkstyle inputs by Test for checks in coding package: NoClone

### PR DESCRIPTION
**Pull request by Kietzmann**
#4165

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java [hunk 1]
```diff
@@ -55,7 +57,7 @@ public void testHasClone()
             "60: " + getCheckMessage(MSG_KEY),
             "98: " + getCheckMessage(MSG_KEY),
         };
-        verify(checkConfig, getPath("InputClone.java"), expected);
+        verify(checkConfig, getPath("InputNoCloneClones.java"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java]: `NoCloneClones` sounds a little weird.
Just name it `InputNoClone`.

Comment (Kietzmann) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java]: Done

**Comment by romani** [comment]
please rebase on latest code to resolve TC problem, TC problems are resolved in our master.

**Comment by Kietzmann** [comment]
@romani 
Done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java (hunk 1 above)
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java]: @Kietzmann This item is still not done.
Please rename to `InputNoClone`.

Comment (Kietzmann) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheckTest.java]: Done

**Pull request merged**